src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -19,16 +40,11 @@ public static string RenderPartial(string partialName, object model, string[] vi
                 routeData.Values.Add("controller", "DummyController");
 
                 var controllerContext = new ControllerContext(new RequestContext(httpContext, routeData), new DummyController());
-
-                var viewEngine = new RazorViewEngine
-                {
-                    PartialViewLocationFormats = viewLocations
-                };
-
-                var viewResult = viewEngine.FindPartialView(controllerContext, partialName, false);
+                
+                var viewResult = ViewEngine.FindPartialView(controllerContext, partialName, false);
                 if (viewResult.View == null)
                 {
-                    // TODO: Log lack of view?
+                    LogHelper.Warn(typeof(ViewHelper), "No view found for partial '" + partialName + "'");
```

Comment:
Tiny thing. We could use an interpolated string here - since we're supporting C#7.

Reply:
didn't know if c# 7 was supported, I'll update it.